app/common/locales/en.json

```diff
@@ -840,7 +840,8 @@
             "upload" : "Upload",
             "add_photo": "Add photo",
             "delete_photo": "Delete photo",
-            "replace_image": "Replace image"
+            "replace_image": "Replace image",
+            "error_in_upload": "Your image is too large or has the wrong format, please check and try again!"
```

Comment:
can you provide information about the max image size and the supported formats?

Reply:
Sent a fix for this!